src/librustc/hir/def.rs

```diff
@@ -132,6 +132,7 @@ pub struct Export {
     /// We include non-`pub` exports for hygienic macros that get used from extern crates.
     pub vis: ty::Visibility,
     /// True if from a `use` or and `extern crate`.
+    /// Used in rustdoc.
     pub is_import: bool,
```

Comment:
Can't this be checked by way of `tcx.def_key(export.def.def_id()).parent`? Or by adding a `DefId` of the export itself, and comparing that to `export.def.def_id()`.

Reply:
Yeah, this should be removable with a little refactoring in rustdoc, opened https://github.com/rust-lang/rust/issues/47100.